docs/yaml/functions/add_project_link_arguments.yaml

```diff
@@ -3,7 +3,7 @@ returns: void
 description: |
   Adds global arguments to the linker command line.
```

Comment:
I feel like while we're here it probably make sense to change this to not use the world "global"

Otherwise this is a nice catch

Reply:
Good point, amended to use the same wording as for `add_project_arguments`.